recipes/src/main/java/org/apache/camel/quarkus/update/CamelYamGroupedRecipe.java

```diff
@@ -0,0 +1,55 @@
+package org.apache.camel.quarkus.update;
+
+import org.apache.camel.quarkus.update.yaml.CamelYamlRouteConfigurationSequenceRecipe;
+import org.apache.camel.quarkus.update.yaml.CamelYamlStepsInFromRecipe;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.yaml.YamlIsoVisitor;
+import org.openrewrite.yaml.tree.Yaml;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ * Grouping recipe for all Yaml recipes.
+ * Recipes are registered in the list (order is preserved) and all of them are executed.
+ */
+public class CamelYamGroupedRecipe extends Recipe {
+
+    List<Recipe> groupedRecipes = Arrays.asList(
+            new CamelYamlStepsInFromRecipe(),
+            new CamelYamlRouteConfigurationSequenceRecipe()
+    );
+
+    @Override
+    public String getDisplayName() {
+        return "Camel Yaml recipes for following changes:\n"
+                + groupedRecipes.stream().map(r -> r.getDisplayName()).collect(Collectors.joining("\n"));
+    }
+
+    @Override
+    public String getDescription() {
+        return "Camel Yaml recipes for following changes:\n"
+                + groupedRecipes.stream().map(r -> r.getDisplayName()).collect(Collectors.joining("\n"));
+    }
+
+    @Override
+    public TreeVisitor<?, ExecutionContext> getVisitor() {
```

Comment:
TODO The same as properties recipe.
There should be a validation, that project contains camel dependencies.
Unfortunately I don't see a way how to validate e.g. file name because it can be anything